examples/setup_config.toml

```diff
@@ -8,27 +8,33 @@ ROOT_DIR = "D:/"
 # user-defined model folder name (naming convention: ModelYear_ModelVersion_Project_Scenario)
 MODEL_FOLDER_NAME = "TM2.2.1.3_clean_setup"
 
-# location of the networks
+# location of the networks support files
 # this includes file such as tolls, transit fares etc.
+# Box: https://mtcdrive.box.com/s/6jf6duborg00d5uqftu35iiynpuzpkbj
 INPUT_NETWORK_DIR = "Z:/projects/MTC/US0024934.9168/setup_model/model_inputs/2015-tm22-dev-sprint-03"
 
 # this is the location/folder for Emme Netorks (built from Lasso) 
+# Box: https://mtcdrive.box.com/s/rs84fqlzkpidn1cdxruucy70500seye5
 INPUT_EMME_NETWORK_DIR = "Z:/projects/MTC/US0024934.9168/setup_model/emme_network"
 
 # location of the populationsim and land use inputs
+# Box: https://mtcdrive.box.com/s/6jf6duborg00d5uqftu35iiynpuzpkbj
 INPUT_POPLU_DIR = "Z:/projects/MTC/US0024934.9168/setup_model/model_inputs/2015-tm22-dev-sprint-03"
 
 # location of nonres inputs
+# Box: https://mtcdrive.box.com/s/6jf6duborg00d5uqftu35iiynpuzpkbj
 INPUT_NONRES_DIR = "Z:/projects/MTC/US0024934.9168/setup_model/model_inputs/2015-tm22-dev-sprint-03"
 
 # this is the location of a previous run folder which has the warm start demand or skims
+# Box: https://mtcdrive.box.com/s/rj0kh56lbqjxhtcmg96ifcc4x6knsmqz
 WARMSTART_FILES_DIR = "Z:/projects/MTC/transit_path_builder_calib/TM2.2.1.2/warmstart"
 
 # this is the location/folder for Emme Netorks 
+# Box: https://mtcdrive.box.com/s/a59dn0o6c3s11tbb65b5z2ww2wc0ixtl
 EMME_TEMPLATE_PROJECT_DIR = "Z:/projects/MTC/US0024934.9168/setup_model/emme_23_project_template"
 
 # github raw link for toml config files
 CONFIGS_GITHUB_PATH = "https://raw.githubusercontent.com/BayAreaMetro/travel-model-two-config/refs/heads/main/2015-tm22-dev-sprint-03"
 
 # travel model two release tag to fetch
-TRAVEL_MODEL_TWO_RELEASE_TAG = "TM2.2.3"
\ No newline at end of file
+TRAVEL_MODEL_TWO_RELEASE_TAG = "TM2.2.4"
```

Comment:
@Ennazus note that we are changing it to use the latest travel model two (CT-RAMP) release.